add a lua script to include short titles on headers (#310)
rstudio/rticles: NEWS.md

rticles 0.15
---------------------------------------------------------------------

> - Improve `jss_article()` format: Short titles can now be provided to headers to escape code/math in section titles, continuation prompt as the correct formatting and skeleton has been updated accordingly (thanks, @statibk, #254, @Freguglia, #294) 

Reviewer: ```suggestion
- Improved the `jss_article()` format: Short titles can now be provided to headers to escape code/math in section titles, the continuation prompt has been corrected (from `R+` to `+`), and the skeleton document has been updated accordingly (thanks, @statibk #254, @Freguglia #294).
```
